test.lua

```diff
@@ -145,36 +145,6 @@ local function add_entry(space_id, uid, email, expiration_time)
     box.space[space_id]:replace{uid, email, expiration_time}
 end
 
--- put test tuples
-local function put_test_tuples(space_id, total)
-    local time = math.floor(fiber.time())
```

Comment:
In this line `Function expiration_process instead of simple deletion` i made a typo. Instead of` expiration_process`, i need to write `process_expired_tuple`

Reply:
Rewrite to `process_expired_tuple`